sherpa/optmethods/__init__.py

```diff
@@ -29,6 +29,12 @@
 
 __all__ = ('GridSearch', 'OptMethod', 'LevMar', 'MonCar', 'NelderMead')
 
+try:
+    import iminuit
+    __all__ = optfcts.add2__all__(__all__, 'Minuit')
```

Comment:
You should be able to remove `optfcts.add2__all__` and just say

    __all__ += ('Minuit', )

and similarly in ` sherpa/optmethods/optfcts.py `